bin/sidecli.re

```diff
@@ -167,7 +167,14 @@ let info_create_transaction = {
 };
 
 let create_transaction =
-    (folder_node, sender_wallet_file, received_address, amount, ticket) => {
+    (
+      folder_node,
+      endpoint,
```

Comment:
This field is not needed, you're already passing `folder_node` you can access the `identity.json` and get the URI there.

Reply:
Addressed in the incoming commit